.github/workflows/rit.yml

```diff
@@ -0,0 +1,68 @@
+name: Rootstock Integration Tests
+
+on:
+  pull_request:
+    types: [opened, reopened]
+    branches:
+      - "**"
```

Comment:
suggestion:

Probably makes sense consider what @bcodesido said in the other [PR](https://github.com/rsksmart/rskj/pull/2701/files#r1745501462).

Reply:
Addressed, removed the pull request event